### DISCUSSION TITLE
요청: translations/{id} fileName 추가

### DISCUSSION THREAD

**Discussion by jun-brro** [sunnybraille/sunnybraille-server#93] [Ideas]
<img width="1387" alt="image" src="https://github.com/sunnybraille/sunnybraille-server/assets/115399447/f6e75940-83fb-4ec1-8c57-b296e558cc44">

이쪽 이미지에서 id와 fileName이 같이 나오면 좋겠습니다.

<img width="367" alt="image" src="https://github.com/sunnybraille/sunnybraille-server/assets/115399447/604fa33b-5b04-42a9-8f19-0c2a735ac501">

이 내용에서 파일 아이콘 밑에 파일명.brf을 표시하려고 합니다. 사이트에서 스크린리더로 본인이 지정한 파일이 잘 변환되었는지 확인시켜주기 위함입니다. translations/ api에서 originalFileName 배출해주셨던 것 그대로 보내주시면 제가 후처리해서 .brf 형태로 보여주게 하겠습니다..!

**Comment by gitchannn** [comment]
그냥 올리면 알림이 안와서 골뱅이로 언급 부탁드립니다!

**Comment by gitchannn** [comment]
확인했습니다! 반영 후에 다시 댓글 남길게요!
